parallel-install/example/example.go

```diff
@@ -30,8 +31,9 @@ func main() {
 
 	log := logger.NewLogger(*verbose)
 
-	if kubeconfigPath == nil || *kubeconfigPath == "" {
-		log.Fatal("kubeconfig is required")
+	if (kubeconfigPath == nil || *kubeconfigPath == "") &&
+		(kubeconfigRaw == nil || *kubeconfigRaw == "") {
+		log.Fatal("either kubeconfig or kubeconfigcontent property has to be set")
```

Comment:
it's `kubeconfigraw` or `kubeconfigcontent`?

Reply:
it should be kubeconfigContent i would say

Reply:
At first I called It `kubeconfigcontent` but later I have spotted Magda mentioning `kubeconfigraw` as a potential candidate to being exposed as a property. I have simply not changed the error message.